rosdep/base.yaml

```diff
@@ -2936,7 +2936,10 @@ libzmqpp3:
   ubuntu: [libzmqpp3]
 linux-headers-generic:
   arch: [linux-headers]
-  debian: [linux-headers-generic]
+  debian:
+    jessie: [linux-headers-3.16.0-4-all]
+    sid: [linux-headers-3.16.0-4-all]
```

Comment:
We generally don't do rules for sid since it's a rolling release, but it would be good to add stretch: https://packages.debian.org/stretch/linux-headers-4.6.0-1-all